packages/web3-eth2-beaconchain/test/schemaMethods.test.ts

```diff
@@ -135,7 +135,7 @@ it('getValidatorBalances', async () => {
     expect(response[0]).toMatchObject(expectedResponse)
 })
 
-it('getEpochCommittees', async () => {
+xit('getEpochCommittees', async () => {
```

Comment:
typo?

Reply:
Sorry [this](https://github.com/ChainSafe/web3.js/pull/3892#discussion_r567209601) got posted, but it was still pending

Reply:
Related [issue](https://github.com/ChainSafe/web3.js/issues/3893)